aws/resource_aws_devicefarm_project.go

```diff
@@ -35,13 +35,6 @@ func resourceAwsDevicefarmProject() *schema.Resource {
 
 func resourceAwsDevicefarmProjectCreate(d *schema.ResourceData, meta interface{}) error {
 	conn := meta.(*AWSClient).devicefarmconn
-	region := meta.(*AWSClient).region
-
-	//	We need to ensure that DeviceFarm is only being run against us-west-2
-	//	As this is the only place that AWS currently supports it
-	if region != "us-west-2" {
-		return fmt.Errorf("DeviceFarm can only be used with us-west-2. You are trying to use it on %s", region)
-	}
```

Comment:
We don't need this check as the API will fail and we can pass the error along. Removing it allows `devicefarm_project` to be used with new regions AWS adds support for, immediately.